changelog.d/flow-70.fixed

```diff
@@ -0,0 +1 @@
+fix: taint signatures do not capture changes to parameters' fields
```

Comment:
Without an example this is probably not understood by users who are the target audience of the changelog...